### PR TITLE
Add --HEAD option to clang-omp

### PR DESCRIPTION
**Pull request by metacollin**
This will use the trunk versions from the github mirror.  This is useful for testing OpenMP with the latest development version of clang.  Currently builds the trunk of 3.7 correctly.

### CODE REVIEW

File: Library/Formula/clang-omp.rb [hunk 1]
```diff
@@ -15,19 +16,35 @@ class ClangOmp < Formula
   depends_on "libiomp"
   depends_on "cmake" => :build
 
-  resource "compiler-rt" do
-    url "https://github.com/clang-omp/compiler-rt/archive/2015-04-01.tar.gz"
-    sha256 "5a8d39ff6ce524e23fae32870f85b18d43f2795da2011d3cbb6b29d471bb27b7"
-  end
+  stable do
```
Comment (DomT4) [inline Library/Formula/clang-omp.rb]: `stable do` and `head do` blocks should include everything related to `stable` and `head`, including the `url` and `sha256` (where appropriate).

Comment (metacollin) [inline Library/Formula/clang-omp.rb]: Whoops, fixing now.  I was under the false impression those were shared, or 'global'.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by dunn** [comment]
The test bot is having trouble merging this PR into master. Could you rebase it? Thanks!

**Pull request closed without merging**